internal/common/common.go

```diff
@@ -93,6 +94,34 @@ const NMDebugModeConf = `
 domains=ALL:DEBUG
 `
 
+func GetIgnoredValidations(validationsJSON string, validationType string, clusterID string) ([]string, error) {
```

Comment:
I think we should not provide validationType, you can handle error outside the function, no?

Reply:
I guess it's just for the error message so I can refactor that